dev-docs/source/UserSupport.rst

```diff
@@ -0,0 +1,38 @@
+.. _UserSupport:
+
+============
+User Support
+============
+
+.. contents::
+  :local:
+
+Introduction
+############
+
+As Mantid continues to facilitate cutting-edge scientific research, for an increasing number of Users, the Support side of Mantid is growing more and more important. This can be in many circumstances and through different avenues; therefore, below is detailed our Support Procedures.
```

Comment:
```suggestion
As Mantid continues to facilitate cutting-edge scientific research, for an 
increasing number of users, the support side of Mantid is growing more 
and more important. This can be in many circumstances and through 
different avenues; therefore, below is detailed our support procedures.
```
A few unnecessary capilizations. The second sentence is also a bit of a jumble, I don't think it makes sense.